valohai/distributed.py

```diff
@@ -0,0 +1,68 @@
+import json
+from typing import List, Optional
+
+from valohai import paths
+from valohai.internals.distributed_config import DistributedConfig, Member
+
+
+class Distributed:
+    """
+    Distributed toolkit accessed through `valohai.distributed`.
+    """
+
+    _config: Optional[DistributedConfig] = None
+
+    def is_distributed_task(self) -> bool:
+        # not a property to mimic `is_running_in_valohai`
+        try:
+            return bool(self.config.group_name)
+        except (FileNotFoundError, json.JSONDecodeError):
+            return False
```

Comment:
Hey, can we maybe add a bit that makes this function never fail (e.g. if the distributed config changes and we'd be using an old version of valohai-utils)?

```suggestion
        except (FileNotFoundError, json.JSONDecodeError):
            return False
        except Exception as exc:
            warnings.warn(f"Failed to parse distributed config: {exc}")
            return False
```